### ISSUE TITLE
Content Model National Safety Pages

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1923]
An outline of scope, user needs, goals, content model and open questions are all outlined [in this brief](https://docs.google.com/document/d/1LaYnEi8YCYpCnxDrR7SvAPlz-t9i53XuLSAPnbg9j6Y/edit?usp=sharing)

Upon alginment document design work in #2011

**Comment by kmranjo** [comment]
I'm continuing work on the example, and do not yet have the governance piece written up, so those pieces will go into next week. But this is at least ready for an early review of concepts, goals and structure from design and product teams.

**Comment by kmranjo** [comment]
Example is complete, so moving this to review.

**Comment by kmranjo** [comment]
Presented to product, general consensus to the open questions is to keep the section only to the weather events. Activity and location based content can be removed, relevant information included in the event pages where appropriate, and other resources linked out to.

**Comment by kmranjo** [comment]
In follow-up convo, discussed in parallel to design work also coming up with the defined list of what pages will be created. Current structure aggregates some event types, hides others, etc. Getting at least an initial list of pages would be helpful. Going to create another ticket for that and add it to backlog.

**Comment by colinmurphy01** [comment]
Im assuming that needs to happen before #2011

**Comment by kmranjo** [comment]
No, it can happen in parallel. The decision shouldn't have any impact on design, it will just impact what the final list of pages to create will be, and what conversations will be needed to get alignment on that.
